package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-router-scroll",
-  "version": "0.6.1",
+  "version": "0.7.0",
```

Comment:
@snewcomer 👍. Would you like me to merge and publish?

Reply:
Let me make sure this works again on the two apps we have this branch installed on and I'll write back.

Also perhaps @ryanto could verify this branch works (even though you might not need this feature)....

Reply:
Hey @snewcomer I just tried this out on our test suite. I get a lot of test failures, all related to: https://github.com/dollarshaveclub/ember-router-scroll/issues/72

#72 has always been an issue for us, but we skip the tests in our suite where it happens. This PR makes 15 additional tests start failing with the `afterContentPaint` error.

Reply:
@yowainwright tested and g2g!